.github/workflows/lighty-rnc-app/simulator/Dockerfile

```diff
@@ -2,13 +2,12 @@ FROM alpine/git as clone
 WORKDIR /netconf-simulator
 RUN git clone https://github.com/PANTHEONtech/lighty-netconf-simulator.git -b 17.0.1
 
-FROM maven:3.8.5-openjdk-17-slim as build
+FROM maven:3.8-eclipse-temurin-17-alpine as build
 WORKDIR /lighty-netconf-simulator
 COPY --from=clone /netconf-simulator/lighty-netconf-simulator /lighty-netconf-simulator
 RUN mvn -B install -DskipTests
 
-FROM alpine:latest
-RUN apk add openjdk17-jre
+FROM maven:3.8-eclipse-temurin-17-alpine
```

Comment:
Why not use here image only with Java? Maven is not required here, and the image with only Java is 4 times smaller. 
https://hub.docker.com/layers/library/eclipse-temurin/17-jre-alpine/images/sha256-eec36bb66392676c32704aaec4a993d1308aee3b7725705be78c6737c4ada46d?context=explore

Reply:
Yes :D